### PR TITLE
Feat: Basic Routers added

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change logs
- Added pages
- Added Basic Routes to these pages

## Screenshots
image
image

## Preview Link
https://deploy-preview-1--ripple-store-app.netlify.app/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import {
+	Home,
+	Cart,
+	Wishlist,
+	Products,
+	ProductDetails,
+	Profile,
+	Authentication,
+} from "./pages/index";
```
Comment (ravikumar0403) [inline src/App.jsx:9]: `/index` is not required. If the directory has index.js then it directly attempts import from index

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+export default function App() {
+	return (
+		<div className="App">
```
Comment (ravikumar0403) [inline src/App.jsx:13]: This classname can be removed. I see that you are not using it

**Review by ravikumar0403 — APPROVED** [review]
Looks good, I have left a couple of sanity comments

**Review by yashpatel1797 — APPROVED** [review]
I agree with Ravi's suggestion else everything look good.

**Review by bharati-21 — COMMENTED** [review]
Nice idea to create all the routes first! 
Looks good to be merged.

File: src/App.jsx (hunk 1 above)
Comment (sonishreyas) [reply src/App.jsx:9]: Thanks @ravikumar0403 will make that chnage

**Pull request merged**